exercises/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,11 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return numbers(first[0, 2]); // complete this statement
 }
 
 function last(arr) {
-  return; // complete this statement
+  return names(last("Joe")); // complete this statement
```

Comment:
Similarly here, you can get the last item in the array by indexing the length of the array minus one:
```
  return arr[arr.length - 1]; // complete this statement
```

Reply:
Ok, cool. Thank you...